generators/server/templates/src/main/java/package/config/SecurityConfiguration.java.ejs

```diff
@@ -237,7 +237,7 @@ public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
             .featurePolicy("geolocation 'none'; midi 'none'; sync-xhr 'none'; microphone 'none'; camera 'none'; magnetometer 'none'; gyroscope 'none'; speaker 'none'; fullscreen 'self'; payment 'none'")
         .and()
             .frameOptions()
-            .deny()
+            .sameOrigin()
```

Comment:
Is it possible to change that depending on the profile? It makes sense to me to use it only in development  profile while on production (where no H2 console should be there) it would be good to disable it.

Reply:
Will this affect CORS or the ability to be an OAuth 2.0 resource server?

Reply:
If yes it won't work right now as currently the policy is stricter or do I get you wrong?

Reply:
@atomfrede : Yes, good idea; I've added some code to only enable it for the dev profile. Let me know if you see any further issues. :smile: